packages/ui5-info/package.json

```diff
@@ -32,7 +32,7 @@
     ],
     "dependencies": {
         "@sap-ux/logger": "workspace:*",
-        "axios": "1.6.1",
+        "axios": "1.6.5",
```

Comment:
why did you updated the `axios` package?

Reply:
removed